packages/rest-typings/src/v1/customUserStatus.ts

```diff
@@ -35,14 +36,55 @@ const CustomUserStatusListSchema = {
 
 export const isCustomUserStatusListProps = ajv.compile<CustomUserStatusListProps>(CustomUserStatusListSchema);
 
+type CustomUserStatusCreateProps = { name: string; statusType?: UserStatus };
+
+const CustomUserStatusCreateSchema = {
+	type: 'object',
+	properties: {
+		name: {
+			type: 'string',
+		},
+		statusType: {
```

Comment:
nullable?

Reply:
This is a required field (as you can check [here](https://github.com/RocketChat/Rocket.Chat/blob/5c18a1ff3af537de3a8af9e3536353e578069bea/apps/meteor/app/user-status/server/methods/insertOrUpdateUserStatus.ts#L68)). Since this is upposed to be break, I believe we can fix this (make it required even though it didn't use to be)